### ISSUE TITLE
Question: laravel-mix

### ISSUE THREAD

**Issue by ekandreas** [creativetimofficial/vuetify-material-dashboard#1]
If I would like to use a more standard frontend compiler like laravel-mix (webpack wrapper)? Do you have any directions or tips making the conversion?

**Comment by dragosct10** [comment]
Hi, @ekandreas! Thank you for using our products. Unfortunately, we didn't have any interaction with laravel-mix until now, so we can't give any advice on how to make the conversion, but look here a [link](https://www.sitepoint.com/use-laravel-mix-non-laravel-projects/) and maybe this will help you. 

Regards, 
Dragos

**Comment by ekandreas** [comment]
Laravel-Mix is decoupled from Laravel. It is a webpack wrapper that I use in WordPress and other projects that need webpack. 

But I find your design nice...

Med vänlig hälsning, Andreas Ek, 070-3032203

**Comment by darylphuah** [comment]
Using a clean laravel install, I've tried to add vuetify-material-dashboard/src/main.js to webpack, but it will not compile. I get an error. 

`TypeError: Cannot read property 'startsWith' of undefined`

So I looked at the package.json and added the same dependencies and devDependencies, and will instead get this error: 

`ERROR in ./<path>/router/index.js Module build failed (from ./node_modules/babel-loader/lib/index.js): Syntax Error: Support for the experimental syntax 'dynamicImport' isn't currently enabled (22:29):`

I've tried to add @babel/plugin-syntax-dynamic-import to the package.json with no luck.
